### ISSUE TITLE
How to purchase / get one produced?

### ISSUE THREAD

**Issue by colemickens** [randomplum/sdrewire#3]
Hi.

How can I purchase one, or get one produced?

Thanks! And thanks for the project(s)!

**Comment by randomplum** [comment]
Hi,

I’m currently planning a small production run for end of November (chip shortage is a pain…)
I’ll probably sell them through tindie.com <http://tindie.com/>.
I’ll let you know as soon as they’re available.

**Comment by samueldr** [comment]
Are we talking fully assembled, or only the board?

(Subscribed to this issue, assuming this is where you'll make people aware.)

**Comment by randomplum** [comment]
Fully assembled and including the JST SH to 0.1" cable

**Comment by colemickens** [comment]
Hi @randomplum, I hope things are going well for you. I realize that the supply shortage issues are still very much in effect, but I am again finding myself regularly wishing that I had one of these. I'm not sure what the BOM+assembly (+cost of your time) comes out to, but I suspect I'd be willing to pay a pretty premium on that in order to acquire a few of these.

**Comment by randomplum** [comment]
Hi @colemickens ,

I'm well, hope same for you. Great timing. I just got the v1.1 pcb panels and will most likely produce a first batch of 20pcs by the end of week.

![image](https://user-images.githubusercontent.com/58212048/153862084-e2147d59-4240-48fc-b377-f7fe3c49b6ea.png)

Will keep you posted, the price will most likely be ~65$ plus shipping from Switzerland.

**Comment by jwessel** [comment]
I am interested in purchasing one of the fully assembled boards as well. I took a look at http://tindie.com/ but didn't find any available. 

Thanks for all your work on this project.

**Comment by gamelaster** [comment]
Joining the interest

**Comment by bzvbzv** [comment]
Hello, Thank you for the great project. 
What's the status of the project now? Are there any finished board which is tested ok?

**Comment by gamelaster** [comment]
I purchased SDWireC from: https://www.tindie.com/products/badgerdnl/sdwire-usb-c-sd-card-reader-sd-mux/  . Also it is possible to buy SDwire from 3mbed.com

**Comment by colemickens** [comment]
That's not really the same thing, though.

**Comment by randomplum** [comment]
Hi All,

Just a quick update, between problems in sourcing chips and getting fake/broken ones. I've put this project on hold last year. Unfortunately it looks like the GL3224 (the sd card reader chip) availability did not get any better. I'm currently evaluating CH565W as a replacement, but It's not a high priority project for me at the moment.

**Comment by gamelaster** [comment]
@colemickens of course, but just wanted to show available alternatives, having sdrewire would be best, but since it is still not available, both SDWire and SDWireC are quite good.

@randomplum good luck with the evaluating CH565W, looking forward :)

**Comment by zzxxtt** [comment]
@randomplum CH565W can only support UHS-I SDR50, it can not run at the full speed UHS-I at SDR104。Maybe you can try RTS5306e. It is easier to buy and can support full speed.

**Comment by InfraredRidingHood** [comment]
> Hi [@colemickens](https://github.com/colemickens) ,
> 
> I'm well, hope same for you. Great timing. I just got the v1.1 pcb panels and will most likely produce a first batch of 20pcs by the end of week.
> 
> ![image](https://user-images.githubusercontent.com/58212048/153862084-e2147d59-4240-48fc-b377-f7fe3c49b6ea.png)
> 
> Will keep you posted, the price will most likely be ~65$ plus shipping from Switzerland.


Hello,

I'm wondering if you will be posting the V1.1 files and if it's worth my time trying to assemble some of these boards?

Thanks, Stephen.
